src/index.html

```diff
@@ -0,0 +1,151 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="https://interns2024.pages.dev/assets/Space.ico"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Goldman:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./styles/styles.css" />
+    <meta
+      name="Description"
+      content="Embark on a cosmic journey through the innovation showcased by HackerRank Interns 2024. Explore their achievements, projects, and internship experiences in this space-themed universe."
+    />
+    <meta
+      name="Keywords"
+      content="HackerRank Interns 2024, Galactic Interns, Space-themed Showcase, Internship Achievements, Cosmic Projects, Interstellar Experiences, Innovation Galaxy, Stellar Talents, Coding Stars, Virtual Universe, Cosmic Adventure, Intern Profiles, HackerRank Class of 2024, Intergalactic Journey, HackerRank Universe"
+    />
+    <title>
+      Help Hari take interns to HackerSpace! HackerRank is in danger
+    </title>
+    <meta
+      name="title"
+      content="Help Hari take interns to HackerSpace! HackerRank is in danger"
+    />
+    <meta
+      name="description"
+      content="Aliens have invaded Earth and HackerRank is in danger. Help Hari take hackers to HackerSpace."
+    />
```

Comment:
**suggestion:** We've defined the description meta tag twice. Let's use a single one?